hack/vendor.sh

```diff
@@ -1,28 +1,37 @@
 #!/usr/bin/env bash
 
-# This file is just wrapper around vndr (github.com/LK4D4/vndr) tool.
-# For updating dependencies you should change `vendor.conf` file in root of the
-# project. Please refer to https://github.com/LK4D4/vndr/blob/master/README.md for
-# vndr usage.
+# This file is just wrapper around 'go mod vendor' tool.
+# For updating dependencies you should change `vendor.mod` file in root of the
+# project.
 
 set -e
 
-if ! hash vndr; then
-	echo "Please install vndr with \"go get github.com/LK4D4/vndr\" and put it in your \$GOPATH"
-	exit 1
-fi
+cat > go.mod << EOF
+module github.com/moby/moby
+
+go 1.17
+EOF
+
+cat > hack/make/.resources-windows/go.mod << EOF
+module github.com/docker/docker/autogen/winresources/dockerd
```

Comment:
For this I can open an issue to implement goversioninfo here like it has be done in https://github.com/docker/cli/pull/3310 so you would not need to add this hack. WDYT @thaJeztah?

Reply:
Yes, would be great if we could used the same tool.

I'm ok with a temporary hack like this (if it works in the meantime)

Reply:
I think it'd be good as a follow-up, but doesn't seem like a blocker...